cypress/support/step-definitions/animated-menu-button-steps.js

```diff
@@ -8,6 +8,10 @@ When('I trigger Animated Menu Button preview', () => {
   animatedMenuButtonPreview().trigger('mouseover');
 });
 
+When('I trigger Animated Menu Button preview in noIFrame', () => {
```

Comment:
```suggestion
When('I hover over on Animated Menu Button preview in noIFrame', () => {
```

Reply:
`Hover on` just means that we make `hover on` element, instead of `trigger` means we are opening it.

Reply:
`trigger` isn't a good term for BDD, better will use `open`

Reply:
updated